src/Components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -123,6 +132,23 @@ const AdvancedSearch: React.FC<IAdvancedSearchProps> = (props) => {
                             }
                         }}
                     />
+                    <div>
```

Comment:
Where's the spacing above the buttons? Seems like the root of the modal needs a stack to space the buttons off from teh list control.
![image](https://user-images.githubusercontent.com/57726991/189770597-0655dfc6-54ef-4566-8d22-decec661c42f.png)